app/templates/components/widgets/forms/ticket-input.hbs

```diff
@@ -11,11 +11,11 @@
       </div>
     {{else if (eq ticket.type 'donation')}}
       <div class="two fields">
-        <div class="field">
-          {{input type='number' name='min_price' placeholder=(t 'Min') value=ticket.minOrder}}
+        <div class="field {{if (eq (donation-valid ticket.minPrice ticket.maxPrice) false) 'error'}}" >
```

Comment:
No helper needed for this. Define a computed property inside the ticket-input component to do the validation on `ticket.minPrice` and `ticket.maxPrice`

Reply:
@niranjan94 Okay.